### PR TITLE
public_site: v2

### LINKED ISSUE #565: Marketing Site
## Goals

- [ ] A public facing marketing site that communicates our principles and reflects our design
- [ ] Links to more information for a user's desired platform
- [ ] Somewhere our blog could ultimately end up
- [ ] Some interactivity where you can sign up for the waitlist or something of the sort (@Parth Y/N/M?)

### Inspiration

- https://www.madhive.com has a great website regarding use of color and modern styling
- https://tbtc.network less good, but still represents some of the ideas we're going for
- https://www.hashicorp.com more tech-ey but I think HashiCorp makes good use of limited color
- https://bear.app not representative of our design, but has a decent layout for a multi-device app website
  - Some of these images would be hard to generate without full toolchains setup
  - @SmailBarkouch / @Parth  to provide android shots
  - @raayan to provide Apple shots
  - @tvanderstad to provide Windows shows

When in doubt think [Dieter Rams (where applicable 😆)](https://www.vitsoe.com/us/about/good-design)
> Good design is as little design as possible

### Wording

I think @tvanderstad has more of the opinions around wording...

### General Guidelines

- Limited color, mainly black & white with some accenting (similar to how our UIs look)
  - We're using #5274B4 for a few of the app icons, wouldn't be bad to use that as the accent color here, but maybe we can pick a different one of our _colors_ for this
- A monospaced font for the "big stuff" company name and subject lines
- I think we might need some interactivity e.g. an HTTP request to _register for the launch_ (maybe we could do something with WASM here? Not sure if it's there yet though)

### PR DESCRIPTION
**Pull request by Parth**
There are subtle lil animations and tooltips that aren't captured in these screenshots. But to run the site you just need `python3 -m http.server`, or `python -m SimpleHTTPServer`.

Just plain html + css. No deps on anything except google fonts and font awesome. Works well on all screen sizes (3 layouts impl'd). Line count is inflated by the fonts.

tired of working on this for the time being, some things I'd like to do at some point in the future:
+ [ ] a personalized experience based on platform
+ [ ] some more interesting animations for the security section.
+ [ ] minification / bundling (site already has ridiculously low load times)
+ [ ] adaptive favicon

image
imageimage
imageimage

closes #565

### CODE REVIEW

**Comment by plv** [comment]
Could you add screenshots of the site to the PR desc?

File: public_site/index.html [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+			<span class="left">
+				<div class="centered">
+					<div class="content">
+						<h1 id="text">LOCKBOOK</h1>
+						<h3>The best place to store and share thoughts</h3>
+						<p>
+							Privacy shouldn’t be a compromise. That’s why we made Lockbook, a companion for recording
+							thoughts on all your devices. Record, sync, and share your notes with apps engineered to
+							feel like home on every platform. We collect no personal information and encrypt your notes
+							so even we can’t see them. Don’t take our word for it: Lockbook is 100% open-source.
```
Comment (steverusso) [inline public_site/index.html:27]:
```suggestion
							Privacy shouldn’t be a compromise. That’s why we made Lockbook, a secure companion to record, sync, and share your thoughts with apps engineered to
							feel like home on every platform. We collect no personal information and encrypt your notes
							so even we can’t see them. Don’t take our word for it: Lockbook is 100% open-source.
```
It felt a bit redundant / awkward at certain parts. This is just a rough simplification that can probably be refined further.

**Review by steverusso — CHANGES_REQUESTED** [review]

**Review by steverusso — COMMENTED** [review]
Looks good to me

**Comment by plv** [comment]
I have myriad design nits (will list below) but otherwise I think this is a good start. Also, I don't know how much modern web development matters for landing pages but there's some improvements that can be made there, but again I won't touch on that for now.

w/r/t design:
1) For the first screen (black background):
* There's a lot of empty space going on here to the point where the page looks like it hasn't finished loading.  Off the top of my head we can take a look at putting a Lockbook logo in the top left corner and perhaps a header bar with links to the various sections. If we do this, then it would be good to replace the "Lockbook" text in the white box with the subheading ("The best place to store and share thoughts").
* top and bottom padding in the white box is much higher than the left padding. Would increase left padding by a few px so that the `LOCKBOOK` is ever so slightly more indented
* Where is the download links / call to action at?
* Going from black background to a softer dark grey  is more visually appealing 

2) For the red screen:
* The platform icons should be left-aligned. It looks jarring because it's the only center-justified thing onscreen.
* I feel like our screenshots could be better here, but I also think that the apps' designs could be improved. Completely separate conversation.

3) For the blue "honest" screen:
* The shade of blue doesn't look good with the colors we use for the "check" and "x" icons. Would do us good to change the shade. If you ask me personally I think the entire site should go with a different color palette, but again this is a separate conversation.
* Technically we make our money by selling storage, not a note taking app
* Again, centered prices look weird. Left-align them, maybe even add a "Pricing" subheader
* Would also left-align the text in the first column of the table, but I'm less sure about this one so I'd have to see it to make a call

4) The "developer friendly" screen:

* In the future if we have communities on Discord, Stack Overflow, Github issues, etc. it would be good to have buttons and calls to action here
* Going from black background to a softer dark grey  is more visually appealing

**Comment by plv** [comment]
Oh, I didn't realize you already merged this lol. Keeping my comment as notes for future iterations

**Pull request merged**